tests/MongoDB.Driver.Tests/Specifications/crud/AggregateTest.cs

```diff
@@ -28,15 +27,15 @@ public class AggregateTest : CrudOperationWithResultTestBase<List<BsonDocument>>
         private List<BsonDocument> _stages;
         private AggregateOptions _options = new AggregateOptions();
 
-        public override bool CanExecute(ClusterDescription clusterDescription, BsonDocument arguments, out string reason)
+        public override void SkipIfNotSupported(BsonDocument arguments)
         {
-            var version = clusterDescription.Servers[0].Version;
-            reason = string.Format("Server must be at least 2.6.0. Current server is {0}.", version);
-            return !(version < new SemanticVersion(2, 6, 0) &&
-                ((BsonArray)arguments["pipeline"])
-                    .Cast<BsonDocument>()
-                    .Last()
-                    .Contains("$out"));
+            if (((BsonArray)arguments["pipeline"])
```

Comment:
This method threw the error in `crud` spec tests, since I changed `MongoClient` to `DisposablemongoClient`. Because of it `Server.Version` had `null` value periodically.